jsonmodels/builders.py

```diff
@@ -11,12 +11,15 @@
 
 class Builder(object):
 
-    def __init__(self, parent=None, nullable=False):
+    def __init__(self, parent=None, nullable=False, has_default=False,
```

Comment:
You could use `NotSet = object()` marker above and accept only `default=NotSet` (just like below you've done). With this `has_default` could be defined as

```
@property
def has_default(self):
    return self.default is not NotSet
```
so less arguments, code will not desynchronize and you will keep the readability

Reply:
fixed